### PR TITLE
[WIP] Create json output option

### PR DESCRIPTION
**Pull request by acornard**
Add option to get json, csv and yaml output.
Relates to #24

### CODE REVIEW

File: main/gridinit_cmd.c [hunk 1]
```diff
@@ -282,6 +283,62 @@ dump_as_is(FILE *in_stream, void *udata)
...
+print_as_json(gchar *status, gchar *start, gchar *error)
+{
+	gchar header[] = "  {\n";
+	gchar footer[] = "  }";
+	gchar tab[] = "    ";
+
+	fprintf(stdout, "%s", header);
+	fprintf(stdout, "%s\"status\": \"%s\",\n", tab, status);
+	fprintf(stdout, "%s\"start\": \"%s\",\n", tab, start);
+	fprintf(stdout, "%s\"error\": \"%s\",\n", tab, error);
```
Comment (vincent-legoll) [inline main/gridinit_cmd.c]: Are you sure JSON accepts trailing commas ?
```
vince@envy:~$ echo '[1,2,3]'|jsonlint-php 
Valid JSON
vince@envy:~$ echo '[1,2,3,]'|jsonlint-php 
Parse error on line 1:
[1,2,3,]
------^
Expected one of: 'STRING', 'NUMBER', 'NULL', 'TRUE', 'FALSE', '{', '[' - It appears you have an extra trailing comma
```

File: main/gridinit_cmd.c [hunk 2]
```diff
@@ -282,6 +283,62 @@ dump_as_is(FILE *in_stream, void *udata)
...
+			start = NULL;
+			(void)unpack_line(line, &start, &code);
+
+			if (dump_args) {
+				if (code==0 || code==EALREADY)
+					dump_args->count_success ++;
+				else
+					dump_args->count_errors ++;
+			}
+			if (!first)
```
Comment (vincent-legoll) [inline main/gridinit_cmd.c]: same as above comment, in addition your first looks like a boolean...

**Review by vincent-legoll — CHANGES_REQUESTED** [review]
But when you'll add YAML output, you will be allowed to let those trailing commas appear in output... ;-)

You may want to add some tests for your new-style output

**Comment by acornard** [comment]
Feature add on PR #34

**Pull request closed without merging**